app/controllers/users_controller.rb

```diff
@@ -0,0 +1,35 @@
+class UsersController < ApplicationController
```

Comment:
can you please fix the indentation and whitespaces please? it's not a big deal in ruby but it's just more pleasing to the eyes

Reply:
would you please tell me where should I fix it? I can't find any whitespace or wrong indentation